internal/services/recoveryservices/site_recovery_replicated_vm_resource.go

```diff
@@ -108,6 +108,24 @@ func resourceSiteRecoveryReplicatedVM() *pluginsdk.Resource {
 				Optional:         true,
 				ValidateFunc:     azure.ValidateResourceID,
 				DiffSuppressFunc: suppress.CaseDifference,
+				ConflictsWith: []string{
+					"target_availability_zone",
+				},
+			},
+			"target_availability_zone": {
```

Comment:
we also need to add an acceptance test covering this field - see https://github.com/hashicorp/terraform-provider-azurerm/blob/f232d0a1ba5ef5356db2b703387bb11aec2ea691/internal/services/recoveryservices/site_recovery_replicated_vm_resource_test.go#L18-L31 for an example

Reply:
Hi Tom, additional acceptance test add and passed.

$ TF_ACC=1 go test -parallel 30 -v -timeout=20h ./internal/services/recoveryservices -run="TestAccSiteRecoveryReplicatedVm_zone2zone"
=== RUN   TestAccSiteRecoveryReplicatedVm_zone2zone
=== PAUSE TestAccSiteRecoveryReplicatedVm_zone2zone
=== CONT  TestAccSiteRecoveryReplicatedVm_zone2zone
--- PASS: TestAccSiteRecoveryReplicatedVm_zone2zone (2466.12s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/recoveryservices      2467.984s